packages/client/src/pages/PortalSettings/categories/integration/SingleSignOn/sub-components/AddIdpCertificateModal.js

```diff
@@ -36,6 +36,7 @@ const AddIdpCertificateModal = (props) => {
 
         <TextArea
           className="text-area"
+          id="idpCertificate"
```

Comment:
Rename "idpCertificate" to "idp-certificate"

Reply:
Fiix to "idp-certificate"